modules/Sources/Dependencies/AddressBookClient/AddressBookLiveKey.swift

```diff
@@ -206,11 +214,11 @@ extension AddressBookClient: DependencyKey {
         return syncedContacts
     }
     
-    private static func storeContacts(
+    @discardableResult private static func storeContacts(
```

Comment:
If you take my suggestion to remove the `remoteStore` flag, then in that case the result should not be discardable. The reason for it to be discardable currently is that we know it will be `StoreResult.success` whenever `remoteStore` is false. But that reason will no longer apply.